app/assets/javascripts/application.js

```diff
@@ -14,3 +14,11 @@
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+//= require jquery3
+//= require popper
+//= require bootstrap-sprockets
+
+//= require jquery3
```

Comment:
Same as line `#18`, should be removed :)
Same with lines `#23` and `#19`

Reply:
Okey, thank's :)